core/registration.c

```diff
@@ -1330,7 +1330,6 @@ static uint16_t prv_splitLinkAttribute(uint8_t * data,
 
     index--;
     while (index > *valueStart && data[index] == ' ') index--;
-    if (index == *valueStart) return 0;
```

Comment:
It isn't clear to me why this was removed. It would allow a missing value to be treated as a '\0', REG_ATTR_SEPARATOR, or ' ' depending on what was in the buffer,

Reply:
Will have to revisit this, the issue was that "-f 0" was not accepted (passing a zero as the default format)